sqlite.go

```diff
@@ -96,6 +96,12 @@ var Open sqliteh.OpenFunc = func(string, sqliteh.OpenFlags, string) (sqliteh.DB,
 	return nil, fmt.Errorf("cgosqlite.Open is missing")
 }
 
+// TraceFunc is function called by the driver to report a completed query.
```

Comment:
This doc could use a bit more detail.

A reader might wonder: Does a completed query's duration include the time needed to step over each of the results? If so, is it just the time spent talking to the DB, accumulated step-by-step, or it is the total time elapsed from starting query to last result fetched?

Is the query string potentially sensitive, or is it the version with `?`s in it (which could presumably be safely logged)?